### ISSUE TITLE
Help running it in Windows with Python 3.8.5

### ISSUE THREAD

**Issue by jgerhdz** [OpenInformix/IfxPy#50]
Could you pleas help me to run it in windows?

1. There is no pip install for the Ifxy

- [x] ERROR: Could not find a version that satisfies the requirement IfxPy (from versions: none)
- [ ] `IfxPy> python -m pip install IfxPy
- [ ] ERROR: No matching distribution found for IfxPy`

So I downloaded the 2 *.zip windows files from [here](https://github.com/OpenInformix/IfxPy/blob/master/prebuilt/3x/Win64/IfxPy.zip) put them in the same directory and renamed them
```
`fxPy> ls
\IfxPy
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        8/21/2020   1:52 PM            674 cuc_sql.py
-a----        2/17/2018   8:11 PM         113152 IfxPy.pyd
-a----        2/17/2018   8:11 PM          57760 IfxPyDbi.py`

```
This is my version of Python

`IfxPy> python -VV
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)]`

This is the version of the informix SDC Client
`ibm.csdk.4.50.FC2.WIN.zip`

This is my code.
-a----        8/21/2020   1:52 PM            674 cuc_sql.py

```
`import os

if os.name == 'nt' and hasattr(os, 'add_dll_directory'):
    informixdir = os.getenv('INFORMIXDIR', None)
    if informixdir and os.path.exists(informixdir):
        os.add_dll_directory(os.path.join(informixdir, "bin"))
import IfxPy

connStr = 'CLIENT_LOCALE=en_US.57372;DB_LOCALE=en_US.57372;SERVER=10.10.20.18;SERVICE=20532;DATABASE=unitydyndb;uid=administrator;pwd=cisco'
conn = IfxPy.connect(connStr, '', '')

sql = 'select count(*) from tbl_notifyq'
stmt = IfxPy.exec_immediate(conn, sql)
dictionary = IfxPy.fetch_assoc(stmt)

print('tbl_notifyq: ',  dictionary[''])
 
IfxPy.free_result(stmt)
IfxPy.free_stmt(stmt)

IfxPy.close(conn)`

I get this error
`Exception has occurred: ImportError
DLL load failed while importing IfxPy: The specified module could not be found.
  File "IfxPy\cuc_sql.py", line 7, in <module>
    import IfxPy`
```

![Capture](https://user-images.githubusercontent.com/61958128/90931951-e2665180-e3ba-11ea-9690-2111449d3d5a.JPG)

Could you please help me with my issue?

**Comment by jgerhdz** [comment]
Yes, this is the file I downloaded. "ibm.csdk.4.50.**F**C2.WIN.zip" 

` jsagrera commented on Jan 24

4.10.TC9 (anything with a 'T' in the suffix) is a 32-bit version. Your Python has to match the same architecture as the CSDK libraries.
You should be able to download a Developer edition (free) 64-bit CSDK from https://www-01.ibm.com/marketing/iwm/iwm/web/preLogin.do?source=ifxdl&S_PKG=410FC8dewin&lang=en_US (you may need to create an IBM account). Something 4.10.FC12 or 4.50.FC2 should do fine. (**The 'F' in the name indicates a 64-bit version)**`

But I can uninstall it, could you please let me know which exact name of install file I could use?

**Comment by jsagrera** [comment]
First check the version of the current CSDK you have installed in "C:\Program Files (x86)\Informix Client-SDK". Run the setnet32.exe from "C:\Program Files (x86)\Informix Client-SDK\bin". 
If it is 4.50.FC2, it means you have a 64-bit version. Some how the name of the directories do not match what 4.50 uses.

The uninstall process has changed from version to version. With really old versions of CSDK you can do it from the Windows "Uninstall" control panel applet. New ones (>3.70) there is a "uninstallclientsdk.exe" in the Informix CSDK directory.

But **there is no need to uninstall** the old version. You can have more than one installed. The install will add the new version on top of the current PATH directory and update the ODBC registry definition.
